pkg/edit/complete/generators.go

```diff
@@ -123,19 +122,19 @@ func generateFileNames(seed string, onlyExecutable bool) ([]RawItem, error) {
 		// Full filename for source and getStyle.
 		full := dir + name
 
-		suffix := " "
 		if info.IsDir() {
-			suffix = quotedPathSeparator
+			full += pathSeparator
 		} else if info.Mode()&os.ModeSymlink != 0 {
 			stat, err := os.Stat(full)
 			if err == nil && stat.IsDir() {
 				// Symlink to directory.
-				suffix = quotedPathSeparator
+				full += pathSeparator
 			}
 		}
 
 		items = append(items, ComplexItem{
-			Stem: full, CodeSuffix: suffix,
+			Stem:         full,
+			CodeSuffix:   " ",
```

Comment:
`CodeSuffix: " "` should only apply for non-directories. For directories, don't append a space; this is to allow the user to start another completion to complete the files within the directory.

Reply:
Good point!